sdks/rust/src/protocol/payload.rs

```diff
@@ -2295,7 +2295,7 @@ impl SabrePayloadBuilder {
             }) => {
                 let addresses = vec![
                     compute_contract_registry_address(&name)?,
-                    ADMINISTRATORS_SETTING_ADDRESS.into(),
+                    parse_hex(ADMINISTRATORS_SETTING_ADDRESS)?,
```

Comment:
Anyway that this could be byte array constant?

Reply:
Added the address as a byte array and used it here.